index.html

```diff
@@ -1,45 +1,451 @@
+            <li>
+              <!-- Note: keeping the [...] notation to not diverge too far
+              from the getDisplayMediaSet spec. -->
+              <p>Let <var>constraints</var> be
+                <code>[<var>options</var>.<code>video</code><code>]</code>.
+              </p>
+            </li>
+            <li>
+              <p>If <code>constraints.video</code> is <code>false</code>,
+                return a promise [=reject|rejected=] with a newly
+                [=exception/created=] {{TypeError}}.</p>
+            </li>
+            <li>
+              <p>For each [= map/exist | existing =] member
+                in <var>constraints</var> whose value, <var>CS</var>, is
+                a dictionary, run the following steps:</p>
+              <ol>
+                <li>
+                  <p>If <var>CS</var> contains a member named <code>advanced</code>,
+                    return a promise [=reject|rejected=] with a newly
+                    [=exception/created=] {{TypeError}}.</p>
+                </li>
+                <li>
+                  <p>If <var>CS</var> contains a member whose name specifies a
+                    constrainable property applicable to
+                    <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>,
+                    and whose value in turn is a dictionary containing a member
+                    named either <code>min</code> or <code>exact</code>, return
+                    a promise [=reject|rejected=] with a newly
+                    [=exception/created=] {{TypeError}}.
+                  </p>
+                </li>
+                <li>
+                  <p>If <var>CS</var> contains a member whose name specifies a
+                    constrainable property applicable to
+                    <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>,
+                    and whose value in turn is a dictionary containing a member
+                    named <code>max</code>, and that member's value in turn is
+                    less than the constrainable property's
+                    <a data-cite="screen-capture#dfn-floor-value">floor value</a>,
+                    then let <var>failedConstraint</var> be the name of the
+                    member, let <var>message</var> be either
+                    <code>undefined</code> or an informative human-readable
+                    message, and return a promise [=reject|rejected=] with a new
+                    <code>OverconstrainedError</code> created by calling
+                    <code>OverconstrainedError(<var>failedConstraint</var>,
+                    <var>message</var>)</code>.
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Let <var>requestedMediaTypes</var> be the set of media
+                types in <var>constraints</var> with either a dictionary
+                value or a value of <code>true</code>.</p>
+            </li>
```

Comment:
This looks superfluous. The only type is video, is it not?

Reply:
It's gone now.